### ISSUE TITLE
Remove Weekly date from weekly he

### ISSUE THREAD

**Issue by benjoabrasado25** [twinssbc/Ionic2-Calendar#505]
Hi guys,

I really like the lib. I just would like to ask if It's possible to remove the number from week. If it's possible how?

Here's the image.
![image](https://user-images.githubusercontent.com/32357669/75506481-1cacf280-5a19-11ea-8aa3-a0d8efd6676d.png)

Thank you,

**Comment by twinssbc** [comment]
@benjoabrasado25 Yes, you just need to add **formatWeekViewDayHeader="EEE"** to the HTML.
